### PR TITLE
Implement guardian form

### PR DESCRIPTION
**Pull request by msoucy**
Allows associating a guardian with students.

To add more than one, fill all previous slots, submit, and go back to the page. It's not pretty but it works.

### CODE REVIEW

**Comment by msoucy** [comment]
Closes #36

File: signinapp/admin/users.py [hunk 1]
```diff
@@ -164,53 +170,65 @@ def edit_student_data():
...
-    nested.first_guardian_phone_number.process_data(first_guardian.phone_number)
-    nested.first_guardian_email.process_data(first_guardian.email)
-    if len(guardians) > 1:
-        second_guardian = guardians[1].user
-        nested.second_guardian_name.process_data(second_guardian.name)
-        nested.second_guardian_phone_number.process_data(second_guardian.phone_number)
-        nested.second_guardian_email.process_data(second_guardian.email)
+    for g in user.student_user_data.guardians:
+        nested.guardian.append_entry(g.user)
+    nested.guardian.append_entry()
```
Comment (bot190) [inline signinapp/admin/users.py:188]: This is a bit awkward since it always adds a guardian slot, but we can deal with that in the future.

**Review by bot190 — COMMENTED** [review]
I've pushed some fixes to various issues I found while testing as well.

**Review by bot190 — APPROVED** [review]

File: signinapp/admin/users.py (hunk 1 above)
Comment (msoucy) [reply signinapp/admin/users.py:188]: That was intentional - the point was to always have space for one more. I do the same thing in the guardian list of students

**Pull request merged**